### PR TITLE
Support WaitHandleWait events

### PR DESCRIPTION
**Pull request by verdie-g**
New event introduced in https://github.com/dotnet/runtime/pull/94737

### CODE REVIEW

File: src/TraceEvent/Parsers/ClrTraceEventParser.cs [hunk 1]
```diff
@@ -2157,6 +2186,14 @@ static private TypeLoadStopTraceData TypeLoadStopTemplate(Action<TypeLoadStopTra
         {                  // action, eventid, taskid, taskName, taskGuid, opcode, opcodeName, providerGuid, providerName
             return new TypeLoadStopTraceData(action, 74, 33, "TypeLoad", LoaderTaskGuid, 2, "Stop", ProviderGuid, ProviderName);
         }
+        static private WaitHandleWaitStartTraceData WaitHandleWaitStartTemplate(Action<WaitHandleWaitStartTraceData> action)
+        {                  // action, eventid, taskid, taskName, taskGuid, opcode, opcodeName, providerGuid, providerName
+            return new WaitHandleWaitStartTraceData(action, 301, 39, "WaitHandleWait", Guid.Empty, 1, "Start", ProviderGuid, ProviderName );
```
Comment (brianrob) [inline src/TraceEvent/Parsers/ClrTraceEventParser.cs:2191]: The 5th parameter of the new templates should be `WaitHandleWaitTaskGuid`.

**Review by brianrob — COMMENTED** [review]
Thanks for this contribution.  Overall, looks good - one change noted below.

**Comment by brianrob** [comment]
@verdie-g, can I ask you to fix the merge conflict, since I just merged your other PR.  :)

**Comment by verdie-g** [comment]
@brianrob done!

**Review by brianrob — APPROVED** [review]

**Pull request merged**